projects/developer/src/app/navbar/navbar.component.scss

```diff
@@ -74,7 +73,7 @@ $navbar-item-width: 150px;
 
 .navbar__logo {
     color: var(--white);
-    margin-right: 1.0em;
+    margin-right: 1em;

```

Comment:
Same with this file, a big diff of style changes, when only two style rules were added.